ci/blackify

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -ev
+cd $(dirname $0)/..
+
+black -l 120 .
```

Comment:
I prefer, the default (80)

I know it is an endless debate, but this what pep8 says and it is the default for Black as well, so better use the default and avoid the `-l`. This avoid discussion :smile: